pysat/tests/test_constellation.py

```diff
@@ -280,3 +280,58 @@ def custom1(inst):
         ans = (self.testConst[0].data['doubleMLT'].values ==
                2. * self.testConst[0].data.mlt.values).all()
         assert ans
+
+
+class TestDeprecation():
+
+    def setup(self):
+        """Runs before every method to create a clean testing setup"""
+        warnings.simplefilter("always")
+
+        instruments = [pysat.Instrument(platform='pysat', name='testing',
+                                        sat_id='10', clean_level='clean')
+                       for i in range(2)]
+        self.testC = pysat.Constellation(instruments)
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing"""
+
+        del self.testC
+
+    def test_deprecation_warning_add(self):
+        """Test if constellation.add is deprecated"""
+
+        with warnings.catch_warnings(record=True) as war:
+            try:
+                # initiate function with NoneTypes since function does not
+                # need to run for DeprecationWarning to be thrown
+                # ==> Save time in unit tests
+                _ = self.testC.add(bounds1=None, label1=None, bounds2=None,
+                                   label2=None, bin3=None, label3=None,
+                                   data_label=None)
+            except ValueError:
```

Comment:
Following the try/except methodology from previous unit tests for deprecation warnings

Reply:
Do you need the `_ = ` preceding the test?

Reply:
Also, should we be testing for the correct `ValueError` message being returned?